server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -166,18 +169,19 @@ final private[worker] class StorageManager(conf: RssConf, workerSource: Abstract
   private val fileInfos =
     new ConcurrentHashMap[String, ConcurrentHashMap[String, FileInfo]]()
   private val RECOVERY_FILE_INFOS_FILE_NAME = "fileInfos.ldb"
-  private var fileInfosDb: DB = null
+  private var db: DB = null
```

Comment:
Pls add comment about the storage  sequence of file infos and user shuffle keys here